pkg/apps/apis/apps/v1/legacy.go

```diff
@@ -0,0 +1,7 @@
+package v1
+
+func init() {
+	// TODO: Remove this once the legacy API will be deprecated. We need this now
+	// in order to make the serialization test pass.
+	LegacySchemeBuilder.Register(RegisterConversions)
```

Comment:
@deads2k is this acceptable? drop in/out and the serialization test is passing with this.